### PR TITLE
add aaa-reference.md

### PR DESCRIPTION
**Pull request by kevinzwang**
Copied it from the AAA Reference Google Doc and formatted it for markdown.

The "Example Usage" column on the table is very squished, but I haven't found a solution to that yet. if anyone can help that would be great

### CODE REVIEW

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/autonomous/aaa-reference.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+| Name | Description | Example Usage |
+| --- | --- | ---|
+| end | End the script. Anything after will be ignored. | `end` |
+| exchange | Feeds a cube at exchange height. | `exchange` |
+| intake | Intakes a cube in front of the robot. | `intake` |
+| jump | Jumps to the specified script and continues the current script when finished. (Doesn’t make the robot go up.) | `jump MoveToRScale`
+| move | Move forward or backwards for the specified amount in inches, relative to the current position. | `move 24` |
+| moveto | Move to 1 or more points, sequentially, with an optional last value having a final angle to face towards, all relative to the starting position. | `moveto (12, 0) (36, 12)` <br> `moveto (0, 12) 90` |
+| scale | Place a cube at scale height. | `scale` |
```
Comment (lhmcgann) [inline Robot2018/src/org/usfirst/frc/team199/Robot2018/autonomous/aaa-reference.md]: Able to place on scale at any height? (other alliance could get there first) I know it's not necessarily something to address in this doc, but some other function somewhere else...?

**Comment by lhmcgann** [comment]
I'm just a little confused by how the point system works, because you mention "relative to starting position" and then "relative to current position". How is the robot keeping track of its location? i.e. how does it know where all of these points you will be referencing are? Is there an origin/starting position/reference point to be designated on the field? Relative to current position would work, just trying o figure it out...

File: Robot2018/src/org/usfirst/frc/team199/Robot2018/autonomous/aaa-reference.md (hunk 1 above)
Comment (kevinzwang) [reply Robot2018/src/org/usfirst/frc/team199/Robot2018/autonomous/aaa-reference.md]: So, scale height means that it should maneuver to place it on the scale, no matter if it's tilted or not

**Comment by kevinzwang** [comment]
"relative to starting position" means all the points have the origin at where the robot is when the command is called, and y axis forward and x axis right of the direction that robot is facing

**Comment by lhmcgann** [comment]
Ok that's what I thought but just checking. 👌

**Pull request merged**